osvvm-utility.core

```diff
@@ -0,0 +1,80 @@
+CAPI=2:
+
+name : osvvm:osvvm:utility:2020.05
+
+# The Vivado simulator (xsim) does not currently (2020.1) support the VHDL
```

Comment:
Is TEE the only issue with Viviado?

Reply:
I hadn't investigated past the `tee` error, but it looks like there are several other VHDL 2008 features not supported by the Vivado simulator.

First, it gives the same errors as GHDL regarding the use of `Alert` from the utility functions. Beyond that it reports it doesn't support `context` and `to_string`. Replacing `context` with `use` was an easy work around, but there are a lot of uses of `to_string` in TranscriptPkg and AlertLogPkg that would be more difficult. For example:
```
ERROR: [XSIM 43-4187] File "../src/osvvm/TranscriptPkg.vhd" Line 123 : The "Vhdl 2008 to_string Operator" is not supported yet for simulation.
```
I have no idea if it would be possible to use a local version of `to_string` rather than wait on Xilinx.